x/fungible/types/gas_stablity_pool.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+// GasStabilityPoolAddress returns the address of the gas stability pool
+func GasStabilityPoolAddress(chainID int64) sdk.AccAddress {
```

Comment:
is it necessary to have per chain pool address? 
It feels like a single address could suffice, as it could hold gas ZRC20 balances from different ZRC20 contracts.

Reply:
Yes, it is a good point, I had this in mind for better accounting but we actually already have a different address for all ZRC20.

Reply:
ecbff622